hlf-ca/templates/ingress.yaml

```diff
@@ -14,26 +14,36 @@ spec:
   ingressClassName: {{ .Values.ingress.ingressClassName | quote }}
   {{- end }}
   rules:
-    {{- range .Values.ingress.hosts }}
-    - host: {{ . }}
+    {{- if .Values.ingress.hostname }}
+    - host: {{ include "common.tplvalues.render" ( dict "value" .Values.ingress.hostname "context" $ ) }}
       http:
         paths:
-          - path: {{ $.Values.ingress.path }}
-            pathType: {{ $.Values.ingress.pathType }}
+          {{- if .Values.ingress.extraPaths }}
+          {{- toYaml .Values.ingress.extraPaths | nindent 10 }}
+          {{- end }}
+          - path: {{ .Values.ingress.path }}
+            pathType: {{ .Values.ingress.pathType }}
             backend:
               service:
                 name: {{ include "hlf-ca.fullname" $ }}
                 port:
                   name: http
     {{- end }}
-  {{- if .Values.ingress.tls }}
+    {{- range .Values.ingress.extraHosts }}
+    - host: {{ include "common.tplvalues.render" ( dict "value" .name "context" $ ) }}
+      http:
+        paths:
+          - path: {{ default "/" .path }}
+            pathType: {{ default "ImplementationSpecific" .pathType }}
+            backend:
+                service:
+                  name: {{ include "hlf-ca.fullname" $ }}
+                  port:
+                    name: http
+    {{- end }}
+  {{- if .Values.ingress.extraTls }}
   tls:
-    {{- range .Values.ingress.tls }}
-    - hosts:
-        {{- range .hosts }}
-        - {{ . }}
-        {{- end }}
-      secretName: {{ .secretName }}
-      {{- end }}
+   tls:
```

Comment:
looks like there is twice tls here.

Reply:
Hey @ClementGautier yes thanks for seeing it, it is still a draft, your reviews will be welcome when this PR will be marked ready for review :)